VULNERABILITY_RESPONSE_PROCESS.md

```diff
@@ -125,7 +126,17 @@ PGP key fingerprint = 1218 6272 CD48 E253 9E2D  D29B 66A7 6ECF 9144 09F1
     - c. If disputes arise about whether or when to disclose information about a vulnerability, the Response Team will publicly discuss the issue via IRC and attempt to reach consensus
     - d. If consensus on a timely disclosure is not met (no later than 90 days), the researcher (after 90 days) has every right to expose the vulnerability to the public
 
-## V. Incident Analysis
+## V. Bounty Distribution
+
+- Total availability of XMR bounty can be tracked [here](https://forum.getmonero.org/8/funding-required/87597/monero-bounty-for-hackerone). XMR market values can be found at the various exchanges. See also [Cryptowatch](https://cryptowat.ch/) and [Cryptocurrency Market Capitalizations](https://coinmarketcap.com/).
```

Comment:
Can we avoid coinmarketcap and refer to something not related to cointelegraph/bytecoin? (like livecoinwatch, who also doesn't list obvious scam)

Reply:
Sure. These links were simply meant for people who were otherwise new to cryptocurrency and didn't realize there were markets for such things.